doc/source/data/api/dataset.rst

```diff
@@ -147,9 +147,8 @@ Schema
 .. autosummary::
    :nosignatures:
    :toctree: doc/
-   :template: autosummary/class_without_autosummary_noindex.rst
 
-   Schema
+   dataset.Schema
```

Comment:
is there a way to kip this as `Schema`; the api we want users to use is `ray.data.Schema` (which is as you correctly described only an alias to ray.data.dataset.Schema https://github.com/ray-project/ray/blob/master/python/ray/data/__init__.py#L117)

Reply:
Yep, I added a tilde to remove the `dataset.` from the beginning.